tests/indent/ql/qlpack.yml

```diff
@@ -0,0 +1,7 @@
+---
+library: false
+name: test/test
+version: 0.0.1
+dependencies:
+  codeql/go-all: '*'
```

Comment:
what does this file do?

Reply:
That file tells the CodeQL interpreter where to get the libraries so it does not throw any errors on the `import go` statement. But I can simplify it and remove this file and the import line